tests/test_rpm_verifier.py

```diff
@@ -15,178 +15,164 @@
     ProcessedImage,
     generate_output,
     get_rpmdb,
+    get_rpms_data,
     get_unsigned_rpms,
-    parse_image_input,
 )
 
 
+def test_get_rpmdb(tmp_path: Path) -> None:
+    """Test get_rpmdb"""
+    image = "my-image"
+    mock_runner = MagicMock()
+    out = get_rpmdb(
+        container_image=image,
+        target_dir=tmp_path,
+        runner=mock_runner,
+    )
+    assert mock_runner.call_count == 1
+    assert out == tmp_path
+
+
 @pytest.mark.parametrize(
-    ("processed_images", "expected_print", "expected_failures_exist"),
+    ("test_input", "expected"),
     [
         pytest.param(
-            [ProcessedImage(image="i1", unsigned_rpms=[])],
-            "No unsigned RPMs found.",
-            False,
-            id="No unsigned RPMs + no errors. Do not report failures",
-        ),
-        pytest.param(
-            [ProcessedImage(image="i1", unsigned_rpms=["my-rpm"])],
             dedent(
                 """
-                Found unsigned RPMs:
-                i1: my-rpm
+                libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890
+                python39-twisted-23.10.0-1.el8ap (none)
+                libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890
+                gpg-pubkey-d4082792-5b32db75 (none)
                 """
             ).strip(),
-            True,
-            id="Unsigned RPM + no errors. Report failures",
-        ),
-        pytest.param(
             [
-                ProcessedImage(image="i1", unsigned_rpms=["my-rpm", "another-rpm"]),
-                ProcessedImage(image="i2", unsigned_rpms=[]),
+                "libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890",
+                "python39-twisted-23.10.0-1.el8ap (none)",
+                "libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890",
+                "gpg-pubkey-d4082792-5b32db75 (none)",
```

Comment:
Maybe add another scenario in which we have more than one unsigned RPM, so we can demonstrate that it can return more than one (it can be all-unsigned scenario).

Reply:
The runner returns a long string and split it by lines so we will return a list instead of a string.
Adding signed/unsigned would not make a different here.
We need to check a long string -> returns a list
and an empty string -> returns empty list

Reply:
I meant for `test_get_unsigned_rpms`.

Reply:
Done